File: public/sdk/dotnet/ariana_enterprise_dotnet/examples/FullFlowExample.cs
using ArianaEnterprise;

var ariana = new ArianaEnterpriseClient(new ArianaEnterpriseOptions
{
    ApiKey = Environment.GetEnvironmentVariable("ARIANA_API_KEY") ?? "ari_sbx_xxxxx",
    Environment = "sandbox"
});

await ariana.HealthAsync();
await ariana.Catalog.PushAsync(new []
{
    new ArianaProduct { Sku = "ARI-0001", Name = "Produto Teste", Price = 2299, Stock = 10 }
});
await ariana.Products.UpdateStockAsync("ARI-0001", 8);
await ariana.Products.UpdatePriceAsync("ARI-0001", 2199);

Console.WriteLine("Fluxo Ariana Enterprise .NET executado com sucesso.");

File: public/sdk/java/ariana_enterprise_java/examples/FullFlowExample.java
import br.com.arianamoveis.enterprise.ArianaEnterpriseClient;
import java.util.List;
import java.util.Map;

public class FullFlowExample {
    public static void main(String[] args) throws Exception {
        ArianaEnterpriseClient ariana = ArianaEnterpriseClient.builder()
            .apiKey("ari_sbx_xxxxx")
            .environment("sandbox")
            .build();

        System.out.println(ariana.health());

        ariana.catalog().push(List.of(
            Map.of("sku", "ARI-0001", "name", "Produto Teste", "price", 2299, "stock", 10)
        ));

        ariana.products().updateStock("ARI-0001", 8);
        ariana.products().updatePrice("ARI-0001", 2199);
    }
}
